examples/image_detection.py

```diff
@@ -47,7 +47,7 @@ def _build_ui(self):
                 ):
                     ImageDetection(
                         src="https://placecats.com/300/200",
-                        annotations=(annotation),
+                        annotations=(annotations),
```

Comment:
why do you have parenthesis?  I would definitely recommend having them, but not like you did or what you put in `annotations`. Also the left side of prop, should only be a str/number not a Python object.

Reply:
I don't realize what is happning.  Changed it to use state.

Reply:
What was happening is that the annotation where defined inline inside the vue template with an implicit str conversion. And thankfully the string in the dict were using `'` instead of `"` which allow it to worked. But if that string conversion were to change to use `"` instead it would have break that inline template.